.github/workflows/push_pr_checks_tests.yml

```diff
@@ -218,3 +406,35 @@ jobs:
       E2E_ACCOUNT_ID: ${{ secrets.AC_PROD_E2E_ACCOUNT_ID }}
       E2E_API_KEY: ${{ secrets.AC_PROD_E2E_API_KEY }}
       E2E_LICENSE_KEY: ${{ secrets.AC_PROD_E2E_LICENSE_KEY }}
+
+  check-all-green:
+    name: 🟢 All required checks and tests pass
+    if: always()
+
+    needs:
+      # checks
+    - security
+    - coverage
+    - checks
+    - third-party-notices-check
+    - fmt
```

Comment:
can we call them 
cargo-fmt
cargo-doc
cargo-clippy

Reply:
I have renamed several of the jobs. LMKWYT!